crypto/ec/ecp_nistp521.c

```diff
@@ -1253,6 +1253,20 @@ static void point_add(felem x3, felem y3, felem z3,
     /* ftmp5[i] < 2^61 */
 
     if (x_equal && y_equal && !z1_is_zero && !z2_is_zero) {
+        /* This is obviously not constant-time but it will almost-never happen
+         * for ECDH / ECDSA. The case where it can happen is during scalar-mult
```

Comment:
Do we need to add the same comment to the other curves, too?

Reply:
AFAIK the other curves are immune to this....